lib/unobtainium/world.rb

```diff
@@ -163,14 +164,26 @@ def resolve_options(label, options)
         options.delete("base")
       end
 
+      # we really need :caps and "desired_capabilities" in our options
+      unless options.has_key?(:caps) # rubocop:disable Style/PreferredHashMethods
```

Comment:
the problem is that this hash is an uberhash and does not react to has_key. i tried changing this myself, but it didn't work.